tsfile/src/main/java/org/apache/iotdb/tsfile/file/metadata/statistics/Statistics.java

```diff
@@ -416,14 +416,6 @@ public void setCount(long count) {
     this.count = count;
   }
 
-  public boolean canUseStatistics() {
```

Comment:
remove the  canUseStatistics field in this class

Reply:
fixed